docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc

```diff
@@ -42,7 +42,12 @@ Response:
 
 This aggregation also supports the `precision_threshold` option:
 
+ifdef::asciidoctor[]
+experimental["The `precision_threshold` option is specific to the current internal implementation of the `cardinality` agg, which may change in the future"]
```

Comment:
Could you do `experimental::[` here please?

Reply:
Added with 8d71e10037a3b48efd9ab144ecfcb4006e6fdaa6.